src/dict/transliteration/custom.cc

```diff
@@ -36,7 +36,7 @@ void CustomTransTable::parse( const QString & content )
       continue;
     }
 
-    ins( parts[ 0 ].toUtf8(), parts[ 1 ].toUtf8() );
+    ins( parts[ 0 ].toStdString().c_str(), parts[ 1 ].toStdString().c_str() );
```

Comment:
`QT_NO_CAST_TO_ASCII`

I think the potential problem might be QByteArray doesn't guarantee the ending null byte.

`QString::toUtf8() -> char*`